src/services/cacheConfigUtils.ts

```diff
@@ -203,9 +207,54 @@ export class CacheConfigUtils {
       };
     };
 
+    // Helper for Index entities
+    const indexedEntitiesHelper = () => {
+      let indexedEntities = [];
+      data.feature.indexedCache.indexedEntities.forEach((entity) => {
+        indexedEntities.push('indexed-entity' + ':' + entity);
+      });
+      return indexedEntities;
```

Comment:
![Screenshot from 2022-04-22 00-30-36](https://user-images.githubusercontent.com/97534722/164534198-11aab44c-e27b-460c-b31f-260dd9baf5d1.png)

indexed-entities is getting as an array, instead we need as a object.